### DISCUSSION TITLE
Obsidian Alternative

### DISCUSSION THREAD

**Discussion by xttlegendapi** [streetwriters/notesnook#5952] [General]
obsidian is very cool and popular software but not open source and many people want it to be open source and have web support. maybe is this something you guys should put your focus on. features of obsidian like a clean UI using subtle colors and icons (your UI is more like evernote which is a lot more messy with texts, maybe you can make a mode switch for this) and a notes graph view ,daily notes, and plugin support for community plugins for things like custom data visualizations, Excalidraw, copilot, and a lot more. and maybe also a standard file format, so they are easily storeable/backupable/downloadable and support for other files formats to view.

**Comment by nihalxkumar** [comment]
@xttlegendapi There are many FOSS alternatives to Obsidian. The closest identical one is [Logseq](https://github.com/logseq/logseq)
